patches/0002-Add-Mariner-FIPS-package-upgrade.patch

```diff
@@ -105,8 +105,9 @@ index 1c8c549b281edc..6e3544787124cc 100644
 +		# To reduce image size, CBL-Mariner image doesn't include full ca-certificates: https://github.com/microsoft/CBL-Mariner/issues/3563
 +		# Full certs are expected in a builder-style image (e.g. to fetch dependencies), so install them.
 +		ca-certificates \
-+		# Azure Pipelines expects "useradd", "groupadd", and "su".
++		# Azure Pipelines expects various tools that would be in a buildpack-deps based image.
```

Comment:
Can we add a link to buildpack-deps docs?

Reply:
Done, I reorganized this part in general too to try to make it more direct. (And included a couple line breaks for readability.)